### DISCUSSION TITLE
Welcome to COIN-OR Optimization Suite Discussions!

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/COIN-OR-OptimizationSuite#13] [General]
## 👋 Welcome!
  As you may have heard, Github has been developing a new feature called Discussions, which is now being rolled 
  out in beta to all projects. We envision Discussions as a way of moving the kinds of conversations that currently 
  take place on the old Mailman list serve directly to where development is taking place! To take this thing for a test 
  drive, please post here when you would otherwise post to the list serve. This is a place to connect with other members 
  of our community on topics not suited for opening an Issue. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, feel free to comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xmunoz** [comment]
Hi, I'm Cristina. I not a mathematician, but I used to work at a research institute with @tkralphs building tooling for mathematicians. I have been contributing to the COIN-OR test, build and release system, and occasionally picking up stray community management tasks on the side. I'm a long-time open source contributor, and look forward to helping grow the COIN-OR community!
